styles.css

```diff
@@ -0,0 +1,534 @@
+  color: #344563;
+  font-family: "Poppins", sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+header {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  background-color: white !important;
+}
+
+.container-nav {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: row;
+  color: white;
+  width: 80%;
+  margin: 0 auto;
+  justify-content: space-between;
+  padding: 0 5px;
+}
+
+.nav-bar {
+  display: flex;
+  list-style: none;
+  flex: 0.3;
+  color: white;
+}
+
+.nav-bar ul {
+  display: flex;
+  list-style: none;
+  flex: 1;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.nav-bar a {
+  color: #344563;
+  text-decoration: none;
+  font-size: 15px;
+  font-style: medium;
+}
+
+.hamburger-btn {
+  display: none;
+}
+
+.introduction {
+  padding: 40vh 0;
+  border-bottom-left-radius: 80px;
+  background-color: white;
+  background-image: url(./assets/Rectangle\ 6.png);
+  background-repeat: no-repeat;
+  background-position: absolute;
+  bottom: 9000px;
+  right: 500px;
+  left: 7900px;
+  top: 600px;
+  margin-top: 50px;
+  padding-left: 100px;
+}
+
+.logo-title {
+  font-size: 18px;
+  font-weight: 500;
+  line-height: 15px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #6070ff;
+  margin: 10px;
+  padding: 10px 20px;
+  text-decoration: none;
+}
+
+.greeting {
+  font-style: bold;
+  font-size: 30px;
+  align-content: center;
+  letter-spacing: 0.37px;
+}
+
+.container {
+  width: 90%;
+  margin: 0 auto;
+}
+
+.first-ellipse {
+  background-repeat: no-repeat;
+  position: absolute;
+  left: 40px;
+  top: 110px;
+}
+
+.first_img {
+  position: absolute;
+  top: 100px;
+  right: 20px;
+}
+
+.second_img {
+  position: absolute;
+  top: 100px;
+  right: 450px;
+}
+
+.img_three {
+  position: absolute;
+  top: 500px;
+  left: 150px;
+}
+
+.img_four {
+  position: absolute;
+  right: 400px;
+  top: 500px;
+}
+
+.about-myself .row {
+  display: flex;
+  flex-direction: row;
+}
+
+.connect {
+  font-style: medium;
+  font-size: 10px;
+  color: #7f8cff;
+}
+
+.column {
+  width: 50%;
+  flex: 0 0 50%;
+  margin: 10px;
+}
+
+.social-links ul {
+  list-style: none;
+  flex-direction: row;
+  display: flex;
+  padding-left: 0;
+  margin-left: -5px;
+}
+
+.social-links {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 0;
+  width: auto;
+}
+
+.social-links a {
+  margin: 0 10px;
+  color: black;
+  width: auto;
+}
+
+.ellipse_cut {
+  display: none;
+}
+
+.about-myself {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  background-color: white;
+  border-top-right-radius: 80px;
+  padding: 30px 0;
+}
+
+.social-media ul {
+  list-style: none;
+  flex-direction: row;
+  display: flex;
+  padding-left: 0;
+  margin-left: -15px;
+}
+
+.social-media a {
+  margin: 0 10px;
+  color: black;
+  width: auto;
+}
+
+.resume {
+  color: #396df2;
+  border: 1px solid #396df2;
+  border-radius: 6px;
+  padding: 15px 10px;
+  background-color: white;
+}
```

Comment:
- [ ] Please add button interaction stylings. It should change its color when: Active, Hover,ed and Pressed. Please see the design and make it look like the design. Please don't forget to change the cursor to pointer on hover.